taikun/resource_taikun_access_profile.go

```diff
@@ -282,6 +263,13 @@ func resourceTaikunAccessProfileUpdate(ctx context.Context, data *schema.Resourc
 
 	data.SetId(updateResponse.Payload.ID)
 
+	if data.Get("lock").(bool) {
+		id, _ := atoi32(updateResponse.Payload.ID)
```

Comment:
We should always check the result of atoi.
Here atoi is useless, we already have the id. Btw We can remove Line 264 because the id won't change.